src/utils/got.ts

```diff
@@ -1,9 +1,47 @@
-import got, { ExtendOptions } from 'got'
+import got, { ExtendOptions, TimeoutError } from 'got'
+import { logger } from '../log'
 
 // https://github.com/sindresorhus/got/blob/HEAD/documentation/quick-start.md#options
 const options: ExtendOptions = {
   timeout: {
-    request: 10 * 1000,
+    request: 15 * 1000,
+  },
+  hooks: {
+    beforeError: [
+      (err) => {
+        if (err instanceof TimeoutError) {
+          logger.warn(
+            {
+              err,
+              request: {
+                url: err.request.requestUrl,
+                options: err.request.options,
+                timings: err.timings,
+              },
+            },
+            `Request timed out on ${err.request.options.method} ${err.request.requestUrl}`,
+          )
+        }
+        return err
+      },
+    ],
+    afterResponse: [
+      (response) => {
+        logger.info(
```

Comment:
I first wanted to use `debug` but realized `src/runtime/getPositions.ts` uses logger.debug extensively which can get really noisy if we enable debug logging in prod. We could change the runtime logs to use `trace` and use debug here though, but didn't seem worth it

Reply:
ah right!
Alternatively, we could have different logging "domains" / "components" where we could adjust the verbosity.
We could do this with [bunyan's child](https://github.com/trentm/node-bunyan?tab=readme-ov-file#logchild) feature.

Reply:
ooh, I'll look into this in a follow up